code/modules/jobs/job_types/_job.dm

```diff
@@ -344,7 +344,9 @@
 				back = messenger //Department messenger bag
 			else
 				back = backpack //Department backpack
-
+	if (human.age < AGE_MINOR)
+		var/obj/item/organ/internal/tongue/tongue = H.get_organ_slot(ORGAN_SLOT_TONGUE)
+		tongue.liked_foodtypes = KIDS
```

Comment:
I don't like how this affect the tongue rather than being part of `carbon/get_liked_foodtypes()`. It's less reliable.